src/python_tests_runner/PythonTestRunner.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+using NUnit.Framework;
+
+using Python.Runtime;
+
+namespace Python.PythonTestsRunner
+{
+    public class PythonTestRunner
+    {
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+            PythonEngine.Initialize();
+        }
+
+        [OneTimeTearDown]
+        public void Dispose()
+        {
+            PythonEngine.Shutdown();
+        }
+
+        /// <summary>
+        /// Selects the Python tests to be run as embedded tests.
+        /// </summary>
+        /// <returns></returns>
+        static IEnumerable<string[]> PythonTestCases()
+        {
+            // Add the test that you want to debug here.
+            yield return new[] { "test_enum", "test_enum_standard_attrs" };
+            yield return new[] { "test_generic", "test_missing_generic_type" };
+        }
```

Comment:
I thought you would discover all tests. Are they not displayed in Test Explorer after a run?

Reply:
There is no need to discover all tests.  That would only slow down the normal testing process.  The lists in PythonTestCases are indeed displayed in Test Explorer after a run.